1. Failure: Failure:1 (@tests.R#8) ---------------------------------------------
Failure has been forced


2. Failure: Failure:2a (@tests.R#12) -------------------------------------------
Failure has been forced


3. Failure: Failure:2b (@tests.R#15) -------------------------------------------
FALSE isn't true.


4. Failure: Failure:loop (@tests.R#20) -----------------------------------------
`i` not equal to 2.
1/1 mismatches
[1] 1 - 2 == -1


5. Error: Error:1 (@tests.R#28) ------------------------------------------------
stop
1: stop("stop") at reporters/tests.R:28

6. Error: Error:3 (@tests.R#36) ------------------------------------------------
!
1: expect_message(f(), NA) at reporters/tests.R:36
2: evaluate_promise(object, capture_warnings = FALSE) at /home/muelleki/git/R/testthat/R/expect-output.R:145
3: with_reporter("cache", result <- with_sink(temp, withCallingHandlers(withVisible(code), 
       warning = handle_warning, message = handle_message))) at /home/muelleki/git/R/testthat/R/evaluate-promise.R:56
4: force(code) at /home/muelleki/git/R/testthat/R/reporter-zzz.R:53
5: with_sink(temp, withCallingHandlers(withVisible(code), warning = handle_warning, 
       message = handle_message))
6: withCallingHandlers(withVisible(code), warning = handle_warning, message = handle_message) at /home/muelleki/git/R/testthat/R/evaluate-promise.R:90
7: withVisible(code)
8: f()
9: g() at reporters/tests.R:32
10: h() at reporters/tests.R:33
11: stop("!") at reporters/tests.R:34

testthat results ================================================================
OK: 5 SKIPPED: 2 FAILED: 6
1. Failure: Failure:1 (@tests.R#8) 
2. Failure: Failure:2a (@tests.R#12) 
3. Failure: Failure:2b (@tests.R#15) 
4. Failure: Failure:loop (@tests.R#20) 
5. Error: Error:1 (@tests.R#28) 
6. Error: Error:3 (@tests.R#36) 

